scala/scala.bzl

```diff
@@ -620,6 +620,30 @@ _implicit_deps = {
   "_jdk": attr.label(default=Label("//tools/defaults:jdk"), allow_files=True),
 }
 
+# Single dep to allow IDEs to pickup all the implicit dependencies.
+_shared_resolve_deps = [
+  Label("//external:io_bazel_rules_scala/dependency/scala/scala_library"),
+  Label("//external:io_bazel_rules_scala/dependency/scala/scala_compiler"),
```

Comment:
these are not really implicit dependencies. I think only `scala_library` is. The other ones need to be added explicitly to classpaths generally.